requirements.txt

```diff
@@ -161,7 +161,7 @@ portalocker==2.5.1
 prettytable==3.4.1
 prometheus-client==0.14.1
 prompt-toolkit==3.0.31
-protobuf==3.20.3
+protobuf==4.21.12
```

Comment:
Is it possible to avoid this? Ray requires a version <= 3.20

Reply:
I think so.